plugins/GSdx/GSDevice.cpp

```diff
@@ -45,7 +45,7 @@ GSDevice::GSDevice()
 
 GSDevice::~GSDevice()
 {
-	for(auto &t : m_pool) delete t;
+	for(auto t : m_pool) delete t;
```

Comment:
Yes I read also that `auto` discards by default the `&`, even if (in our case) the iterator `*` operator returns a reference to the element...what a mess.
To keep coherency though, removing it here it's ok.
I think it's no effort to add access by reference (which will replace the current one by value), but, I wonder what changes could lead this to the compiled code...I hope none.